README.rst

```diff
@@ -1,8 +1,121 @@
+
+We want to install our packages in a suitable conda environment.
+The following creates and activates a new environment named ``regolith_env`` ::
+
+        conda create -n regolith_env python=3
+        conda activate regolith_env
+
+Then, to fully install ``regolith`` in our active environment, run ::
+
+        conda install regolith
+
+Another option is to use ``pip`` to download and install the latest release from
+`Python Package Index <https://pypi.python.org>`_.
+To install using ``pip`` into your ``regolith_env`` environment, we will also have to install dependencies ::
+
+        pip install -r https://raw.githubusercontent.com/regro/regolith/main/requirements/run.txt
+
+and then install the package ::
+
+        pip install regolith
+
+If you prefer to install from sources, after installing the dependencies, obtain the source archive from
+`GitHub <https://github.com/regro/regolith/>`_. Once installed, ``cd`` into your ``regolith`` directory
+and run the following ::
+
+        pip install .
+
+Support and Contribute
+----------------------
+
+`Diffpy user group <https://groups.google.com/g/diffpy-users>`_ is the discussion forum for general questions and discussions about the use of regolith. Please join the regolith users community by joining the Google group. The regolith project welcomes your expertise and enthusiasm!
+
+If you see a bug or want to request a feature, please `report it as an issue <https://github.com/regro/regolith/issues>`_ and/or `submit a fix as a PR <https://github.com/regro/regolith/pulls>`_. You can also post it to the `Diffpy user group <https://groups.google.com/g/diffpy-users>`_. 
+
+Feel free to fork the project and contribute. To install regolith
+in a development mode, with its sources being directly used by Python
+rather than copied to a package directory, use the following in the root
+directory ::
+
+        pip install -e .
+
+To ensure code quality and to prevent accidental commits into the default branch, please set up the use of our pre-commit
+hooks.
+
+1. Install pre-commit in your working environment by running ``conda install pre-commit``.
+
+2. Initialize pre-commit (one time only) ``pre-commit install``.
+
+Thereafter your code will be linted by black and isort and checked against flake8 before you can commit.
+If it fails by black or isort, just rerun and it should pass (black and isort will modify the files so should
+pass after they are modified). If the flake8 test fails please see the error messages and fix them manually before
+trying to commit again.
+
+Improvements and fixes are always appreciated.
+
+Before contribuing, please read our `Code of Conduct <https://github.com/regro/regolith/blob/main/CODE_OF_CONDUCT.rst>`_.
+
+Contact
+-------
 
-Docs:
------
-https://regro.github.io/regolith-docs/
+For more information on regolith please visit the project `web-page <https://regro.github.io/>`_ or email Prof. Simon Billinge at sb2896@columbia.edu.
```

Comment:
please fix the link here to be the same as the deleted one which links to the actual docs.